app/DoctrineMigrations/Version20150902154411.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Surfnet\StepupGateway\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+class Version20150902154411 extends AbstractMigration
+{
+    /**
+     * @param Schema $schema
+     */
+    public function up(Schema $schema)
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() != 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+
+        $this->addSql('CREATE TABLE registration (key_handle VARCHAR(255) NOT NULL, public_key VARCHAR(255) NOT NULL, sign_counter INT NOT NULL, PRIMARY KEY(key_handle)) DEFAULT CHARACTER SET utf8 COLLATE utf8_unicode_ci ENGINE = InnoDB');
```

Comment:
I [dove](https://fidoalliance.org/specs/fido-u2f-v1.0-nfc-bt-amendment-20150514/fido-u2f-raw-message-formats.html#registration-response-message-success) in the spec, and the public key is always a 65-byte string, Base64-encoded, resulting in a string that is never more than 88 characters long. I cannot find any requirements of the key handle length other than that, in the registration response, the key handle length is specified using one unsigned byte, ie. 0-255.

Thus, I could resize the public key column to a maximum of 88 characters. Is there value in doing this other than being correct?